apps/condo/domains/onboarding/access/UserHelpRequestFile.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated by `createschema onboarding.UserHelpRequestFile 'userHelpRequest?:Relationship:UserHelpRequest:CASCADE;file:File'`
+ */
+
+const { get } = require('lodash')
+const isEmpty = require('lodash/isEmpty')
+const omit = require('lodash/omit')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const { STAFF } = require('@condo/domains/user/constants/common')
+
+
+const AVAILABLE_TO_UPDATE_USER_HELP_REQUEST_FILE_FIELDS = ['dv', 'sender', 'userHelpRequest', 'deletedAt']
+
+async function canReadUserHelpRequestFiles ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return { createdBy: { id: user.id } }
+}
+
+async function canManageUserHelpRequestFiles ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    if (user.type !== STAFF) return false
+
+    if (operation === 'create') {
+        const userHelpRequestId = get(originalInput, ['userHelpRequest', 'connect', 'id'], null)
+        // all staff users can create temp file without help request connection
+        if (!userHelpRequestId) return true
```

Comment:
Why it so?

Reply:
Same as in all our `File` models.
We check user organization permission when connect object with organization to file (in this case it is `UserHelpRequest` obj)